scripts/style-spec-reference/v8.json

```diff
@@ -1529,7 +1529,8 @@
         "basic functionality": {
           "js": "0.10.0",
           "android": "2.0.1",
-          "ios": "2.0.0"
+          "ios": "2.0.0",
```

Comment:
We maintain this in https://maplibre.org/maplibre-style-spec/

We don't make releases for macOS anymore.

Reply:
Removed the change.